include/lcp-catlist.php

```diff
@@ -117,7 +117,9 @@ private function lcp_categories(){
     if ( is_array($this->lcp_category_id) ){
       return array('category__and' => $this->lcp_category_id);
     } else {
-      if($this->utils->lcp_not_empty('child_categories') && ($this->params['child_categories'] === ('no' || 'false') )){
+      if($this->utils->lcp_not_empty('child_categories') && 
+         (($this->params['child_categories'] === 'no' )
```

Comment:
Is this working for you? Shouldn't there be a `||` between this line and the next?